### PR TITLE
feat: use codes as ids for recommentations codelists and UNCL (EDIFACT)

### PR DESCRIPTION
**Pull request by kshychko**
This PR introduces Rec20 transformer code and the change for the @id value from the name to the common code.
The latest spreadsheet has been added to resources directory and can be used as an input file.
ref #94

### CODE REVIEW

**Comment by kshychko** [comment]
Attaching the output for the reference.
[codelists.zip](https://github.com/uncefact/vocab/files/8957554/codelists.zip)

**Comment by nissimsan** [comment]
Looks great, @kshychko!

File: scripts/transformer/src/main/java/org/unece/uncefact/vocab/TransformationRunner.java [hunk 1]
```diff
@@ -72,6 +70,18 @@ public static void main(String[] args) throws ParseException, IOException, Inval
...
+                transformer = new REC21ToJSONLDVocabulary(inputFileName, outputFileName, prettyPrint);
+                break;
+            case "rec24":
+                transformer = new REC24ToJSONLDVocabulary(inputFileName, outputFileName, prettyPrint);
+                break;
+            case "rec28":
+                transformer = new REC28ToJSONLDVocabulary(inputFileName, outputFileName, prettyPrint);
+                break;
+            case "uncl":
+                transformer = new UNCLToJSONLDVocabulary(inputFileName, outputFileName, prettyPrint);
```
Comment (nissimsan) [inline scripts/transformer/src/main/java/org/unece/uncefact/vocab/TransformationRunner.java:83]: What's UNCL?

**Review by nissimsan — APPROVED** [review]

**Comment by nissimsan** [comment]
Output files are fetched from official publishing.

**Pull request merged**